Bottom: 90aed4c3644c20cc9c7b14490a253b8cefa4a8dd
Top:    811e03ecb2557253e4d730b9dcd0428ce62ae4d7
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-02-20 09:43:42 +1300

Refresh of stream-to-dict

---

diff --git a/setup.py b/setup.py
index 2ad6a62..4b9257d 100755
--- a/setup.py
+++ b/setup.py
@@ -69,7 +69,7 @@ setup(name='testtools',
       zip_safe=False,
       install_requires=[
         'extras',
-        # 'mimetype' has not been uploaded by the maintainer with Python3 compat
-        'python-mimetype',
+        # 'mimeparse' has not been uploaded by the maintainer with Python3 compat
+        'python-mimeparse',
         ],
       )
